.gitignore

```diff
@@ -27,3 +27,10 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+
+# Ignore pre-commit hook configs
+.pre-commit-config.yaml
+
+
+# Ignore inner charts dir
+charts/**/charts
```

Comment:
in what case we need to do it?

Reply:
to ignore the tgz files that are downloaded as part of local dependency build

Reply:
but may we still have subcharts there which are not third-party?

Reply:
ah, right.  Let me change this to `charts/**/charts/*.tgz`. That should do, right?

Reply:
yes